### DISCUSSION TITLE
.NET 9 support?

### DISCUSSION THREAD

**Discussion by martin-honnen** [ikvmnet/ikvm#609] [Ideas]
In November (the 12th or 13th, I think) the .NET 9 SDK will be officially be released to the public, there are already two public release candidates https://dotnet.microsoft.com/en-us/download/dotnet/9.0.

I am wondering whether you intend to update IKVM to specifically target .NET 9 or side by side with .NET 8 (.NET 6 will be phased out this year).

**Comment by wasabii** [comment]
Probably not. Trying to only target LTS releases.
